graylog2-server/src/main/java/org/graylog2/bindings/providers/JestClientProvider.java

```diff
@@ -70,6 +74,10 @@ public JestClientProvider(@Named("elasticsearch_hosts") List<URI> elasticsearchH
                             new AuthScope(hostUri.getHost(), hostUri.getPort(), AuthScope.ANY_REALM, hostUri.getScheme()),
                             new UsernamePasswordCredentials(username, password)
                         );
+
+                        if (!Strings.isNullOrEmpty(username) || !Strings.isNullOrEmpty(password)) {
```

Comment:
Shouldn't this be `&&` instead of `||` so we only add it as preemptive auth host if username **and** password are present?